### DISCUSSION TITLE
Add text inside toggle component

### DISCUSSION THREAD

**Discussion by Carmen-Mercado** [saadeghi/daisyui#1169] [Q&A] [answered]
How can I put text inside toggle component?
I wanna do something like this
<img width="165" alt="Screen Shot 2022-09-21 at 16 04 13" src="https://user-images.githubusercontent.com/30910774/191610087-447110ed-8f24-4949-92be-69d91b52aa6e.png">

**Comment by saadeghi** [comment, marked as answer]
That's the weirdest toggle I've ever seen 😅 but here you are: https://play.tailwindcss.com/SxGTKWw3QN

**Reply by Carmen-Mercado** [reply]
Great!.
Thanks for taking the time to do this.
